database/test/test_services.py

```diff
@@ -78,7 +92,9 @@ def test_create_db(create_db, main_db_params_with_root_user):
         conn.close()
 
 
-def test_populate_koodistot(populate_koodistot, main_db_params):
+def test_populate_koodistot(
+    populate_koodistot, populate_admin_region_geometries, main_db_params
+):
```

Comment:
Here we could perhaps make a separate test test_populate_admin_region_geometries that also uses populate_koodistot and checks that all the geometries are filled.

In that case, we would not need populate_admin_region_geometries here.